docs/src/howto-guides/017_apama_software_management_plugin.md

```diff
@@ -1,70 +1,58 @@
+9. Click the drop-down arrow on the right and check the 10.15.1.0-1::apama radio button, then click **Install**.
+10. Finally, click the **Apply changes** button in the lower right of the panel.
 
-## Download and Install Apama Plugin
+After installation of the support for Apama onto the thin-edge device, the correlator will attempt to start, but will fail initially because there is not yet an Apama project installed. Instructions are given below for installing an Apama project on the device.
 
-> Note: This plugin expects an apama installation on the device which is managed as a System V service. If Apama is not installed or if the System V init script is not installed, follow the Apama installation instructions [here](https://github.com/thin-edge/thin-edge.io_examples/tree/main/StreamingAnalytics#setup-and-configuration) before installing/using the apama plugin.
+## Install Apama artifacts from Cumulocity
 
-Apama plugin is delivered as a debian package which can be downloaded from the releases section of thin-edge.io repository on [GitHub](https://github.com/thin-edge/thin-edge.io/releases).
+Before an Apama project can be installed on the device using the software management feature in Cumulocity, the project files need to be added to the Cumulocity software repository.
 
-Example command for downloading the `0.5.0` version of the plugin for `armhf` architecture (Adjust the version and architecture values as per your requirements):
+There is a naming convention that you need to follow while creating software entries for Apama artifacts in the software repository.
 
-```shell
-curl -LJO https://github.com/thin-edge/thin-edge.io/releases/download/0.5.0/tedge_apama_plugin_0.5.0_armhf.deb
-```
-
-Once the debian package is downloaded, install it with the `dpkg` command:
-
-```shell
-sudo dpkg -i tedge_apama_plugin_0.5.0_armhf.deb
-```
+For Apama projects:
 
-Once the plugin is installed on the device, you can install apama projects and mon files using Cumulocity software management feature.
-
-## Install Apama artefacts from Cumulocity
-
-Before an apama project or a mon files can be installed on the device using the software management feature in Cumulocity, these project files or monitor files need to be added to the Cumulocity software repository first.
-
-There's some naming convention that you need to follow while creating software entries for apama artefacts in the software repository.
-
-For apama projects:
-
-1. Name must be suffixed with `::project` as in `my-demo-project::project`
-2. Version must be suffixed with `::apama` as in `1.0::apama` or just `::apama` if you don't need a version number.
-3. The uploaded binary must be a `zip` file that contains the `project` directory. If a directory named `project` is not found at the root level in the zip, it's considered invalid.
+1. The version must be suffixed with `::Apama` as in `1.0::Apama` or just `::Apama` if no  version number is necessary.
```

Comment:
The earlier version of the `tedge-apama-plugin` that thin-edge team developed was installed at `/etc/tedge/sm-plugins` as `apama` instead of `Apama`. So, if you'd like to replace the existing installations of the old plugin with the new one, you may also wanna name your plugin binary as `apama` and that same name needs to be used as the suffix here. Otherwise both will coexist on machines where the older plugin is already installed. But, if the intent was to keep both side-by-side, then this is fine.

Reply:
Pretty sure we are using lowercase and this is just a doc issue. The package will force-remove the tedge-apama-plugin when it installs

Reply:
The file is lower case. This is a docs error caused by VS code, which I will fix.

Reply:
a8d5ca5